src/index.html

```diff
@@ -11,7 +11,184 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST">
+      <fieldset class="fieldset-style">
+
+        <legend>
+          Personal information
+        </legend>
```

Comment:
```suggestion
      <fieldset class="fieldset-style">
        <legend>
          Personal information
        </legend>
```
gaps only for the same nesting-level elements